README.md

```diff
@@ -1,3 +1,18 @@
 Swagger Typegen
 
-<a href="https://codeclimate.com/github/MatthiasMargot/swagger-typegen/maintainability"><img src="https://api.codeclimate.com/v1/badges/106a3e3f77f51488215a/maintainability" /></a>
\ No newline at end of file
+<a href="https://codeclimate.com/github/MatthiasMargot/swagger-typegen/maintainability"><img src="https://api.codeclimate.com/v1/badges/106a3e3f77f51488215a/maintainability" /></a>
+
+## How to generate the type file
+
+```sh
+$ node cli swagger-input-path swagger-output-path
```

Comment:
Let's do
```suggestion
$ node cli input.json output.ts
```
With file extensions to make it more obvious that the input has to be JSON (will add yaml support too later, there's actually a library in this project called "swagger-parser" which I'm not using any longer currently that converts swagger yaml to json)

Reply:
updated